vyxal/elements.py

```diff
@@ -2585,6 +2585,38 @@ def parity(lhs, ctx):
     }.get(ts, lambda: vectorise(parity, lhs, ctx=ctx))()
 
 
+def parse_direction_arrow_to_integer(lhs, ctx):
+    """Element ¨^
+    (str) -> map characters in `>^<v` to integers
+    """
+    ts = vy_type(lhs)
+    if ts is str and len(lhs) == 1:
+        return {
```

Comment:
Why not manually vectorise for strings here instead? I feel like the intent may be clearer that way.

Reply:
Why not manually vectorise for lists here as well? Because we already have a function for it.

Reply:
Strings aren't lists, though, and I don't feel comfortable relying on vectorise treating strings as iterables.